src/main/java/com/minecolonies/compatibility/tinkers/SlimeTreeCheck.java

```diff
@@ -77,10 +79,8 @@ public boolean checkForTinkersSlimeLeaves(@NotNull final Block block)
      * @param block the block.
      * @return if the block is a slime sapling.
      */
-    @Override
-    @Optional.Method(modid = "tconstruct")
-    public boolean checkForTinkersSlimeSapling(@NotNull final Block block)
+    public static boolean isSlimeSapling(@NotNull final Block block)
     {
-        return block == TinkerWorld.slimeSapling;
+        return new SlimeTreeCheck().checkForTinkersSlimeSapling(block);
```

Comment:
I did not write this, but with an eye on the future actually not.
You would want multiple of these TreeCheckers for special handling of different trees (forestry comes to mind). For now this works but we should replace it with something dynamic in the future.